CMakeLists.txt

```diff
@@ -1,20 +1,22 @@
 cmake_minimum_required(VERSION 2.8.8)
 
+project(googletest-distribution)
+set(GOOGLETEST_VERSION 1.9.0)
```

Comment:
The PR description is specifc to "Fix Cygwin CMake Build". Why are we also inserting a version? Does Gygwin not work without it?

Reply:
I'm not inserting a version. In the first commit I moved these two lines exactly like they were to the beginning of the file because cygwin gets detected by cmake when the project is defined and hence the project definition had to be before using the variable `CYGWIN`. I wrote a description including a link to more explanation under the title "What? - First commit" of my description.

Reply:
My apologies I do not follow, what is the :"first file" . This PR shows me one files changed and the diff shows inserted line 4, +set(GOOGLETEST_VERSION 1.9.0). What I am missing?

Reply:
Oh, nvm, you just moved the 2 lines to the beginning of the file